addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -641,6 +641,7 @@ HtmlField.extractProps = ({ attrs, field }) => {
             maxHeight: attrs.options.maxHeight,
             resizable: 'resizable' in attrs.options ? attrs.options.resizable : false,
             editorPlugins: [QWebPlugin],
+            fieldName: field.name,
```

Comment:
this isn't correct imo. First, this function is about extracting props from the arch (see 95% of the rest of the function). As your field receives the field in props anyway, this could be done in the field itself. Second, the name isn't unique as there might be several occurrences of the same field. The field receives a `id` props for that purpose.

Reply:
I changed the commit to address your comment. Is that what you had in mind?
I added `fieldId: this.props.id` to the `wysiwygOptions` getter on `HtmlField` instead.

Reply:
Yep exactly :-)